src/ert/run_models/everest_run_model.py

```diff
@@ -500,15 +555,14 @@ def _make_evaluator_result(
                 batch_data,
             )
 
-        if self._simulator_cache is not None:
-            for control_idx, (
-                cached_objectives,
-                cached_constraints,
-            ) in cached_results.items():
-                objectives[control_idx, ...] = cached_objectives
-                if constraints is not None:
-                    assert cached_constraints is not None
-                    constraints[control_idx, ...] = cached_constraints
+        for control_idx, (
```

Comment:
I think caching is now always assumed to be on? We do have a flag for in in the configuration, maybe we should kill it.